lib/src/webcrypto/webcrypto.digest.dart

```diff
@@ -98,7 +101,7 @@ abstract class Hash {
   /// ```
   ///
   /// [1]: https://doi.org/10.6028/NIST.FIPS.180-4
-  static const Hash sha1 = impl.sha1;
+  static Hash sha1 = Hash._(webCryptImpl.sha1);
```

Comment:
This is a change in the API, and it could be breaking to people who rely on these being constants.

Reply:
I think we could make them `const` with a tweak like:

```dart
abstract final class Hash {
  const Hash._();
  HashImpl get _impl;
  
  Future<Uint8List> digestBytes(List<int> data) => _impl.digestBytes(data);
  Future<Uint8List> digestStream(Stream<List<int>> data) =>
      _impl.digestStream(data);

  static const Hash sha256 = _Sha256();
}

final class _Sha256 extends Hash {
  const _Sha256() : super._();
  
  @override
  HashImpl get _impl => HashImpl();
}
```